plugins/repository-gcs/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -20,9 +20,10 @@
 grant {
   permission java.lang.RuntimePermission "accessDeclaredMembers";
   permission java.lang.RuntimePermission "setFactory";
+  permission java.lang.RuntimePermission "accessClassInPackage.sun.misc";
```

Comment:
Can you add a comment why it needs to be added?

Reply:
I have removed this permission altogether looks like it's not required (did some manual testing to convince my eyes).

One would say populating the permission list is easy peasy: when you have an access exception you append the list. I don't recall the specific failure and now the failure went away. I would guess it went away when updating some dependcy libs...